video-to-frames/video_frames.py

```diff
@@ -0,0 +1,37 @@
+# DOCUMENTATION
+#   Purpose: Break video files into seperate frames
+#   How to Use:
+#       1 - Set video variable to the of relative path of the video file
+#       2 - (Optional) Change the name of the directory you want to
+#           save the framse to by changing the 'directory_name'
+#   NOTE:
+#      Need to have cv2 installed
+
+
+import cv2
+import os
+
+def FrameCapture(path, directory):
+    vidObj = cv2.VideoCapture(path)
+    count = 0
+    success = True
+
+    while success:
+        success, image = vidObj.read()
+        cv2.imwrite(directory +'/frame%d.jpg' % count, image)
```

Comment:
Is there a way to get the length of the video file, and then pad the count with zeros? Or does that already happen?
So this:  01,02,...,10,11...20
Rather than:  1,10,11,...2,20,21

Reply:
Sounds good